weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -806,8 +853,21 @@ def file_create(self, req: tsi.FileCreateReq) -> tsi.FileCreateRes:
         return tsi.FileCreateRes(digest=digest)
 
     def file_content_read(self, req: tsi.FileContentReadReq) -> tsi.FileContentReadRes:
+        # Must dedupe files
```

Comment:
still a todo?

Reply:
No, sorry those are comments that are intended to say that we're performing deduping in the query. I could clarify those.

Reply:
oh, no it is just a note as to why the query is so wild

Reply:
The comments are updated to be more clear now.